doc/integration.rst

```diff
@@ -160,3 +162,13 @@ Interfaces
 |                         |                         |     | if an external event (interrupt or     |
 |                         |                         |     | debug req) wakes the core up           |
 +-------------------------+-------------------------+-----+----------------------------------------+
+| ``alert_minor_o``       | 1                       | out | Core has detected a fault which it can |
+|                         |                         |     | safely recover from. Can be used by a  |
+|                         |                         |     | system to log errors over time and     |
+|                         |                         |     | detect tampering / attack.             |
++-------------------------+-------------------------+-----+----------------------------------------+
+| ``alert_major_o``       | 1                       | out | Core has detected a fault which cannot |
```

Comment:
Should we document if this signal is a pulse or if it remains high?

Reply:
Yep will do - they both should be pulses (although the major alert pulse might end up being multiple cycles)

Reply:
Thanks.